dw-date-range-input-dialog.js

```diff
@@ -480,7 +480,12 @@ export class DwDateRangeInputDialog extends DwCompositeDialog {
   }
 
   _validateEndDate() {
```

Comment:
`dw-date-range-picker` also validate date in same way, have you validate it it's also has issue or not?